### DISCUSSION TITLE
Drawer -> Tabs -> Stack default route issue.

### DISCUSSION THREAD

**Discussion by markmccoid** [expo/router#347] [Q&A]
I thought I had the structure of expo-router down, but when I added authentication and a drawer,  I couldn't get the default "/" route to resolve unless I used an explicit route.

My directory structure is:

Root
 - (auth)  directory
 - (main) directory
     - (tabs) directory
        - tabOne directory (stack)
        - tabTwo screen

The Root _layout.js is:

The Provider Component redirects to either "/(auth)" or "/(main)"
`  
return (
    <>
      <Provider>
          <Slot />
      </Provider>
    </>
  );
`

The (main) _layout.js defined an initial screen
`
  return (
    <>
      <Drawer
        screenOptions={{ headerShown: false }}
        initialRouteName="(tabs)"
      >
        <Drawer.Screen
          name="(tabs)"
        />
        <Drawer.Screen name="rootscreen" />
      </Drawer>
    </>
  );
`

The (tabs) _layout.js defines the intialRoute as "tabOne", which I want to be a stack.

So, I created a directory called "tabOne" with an index.js, _layout.js and the other screens in the stack. 

My hope was that a redirect to "/(tabs)" would get me to the tabOne index screen, but it says it cannot find the screen.

It works if I either:

1. Make the folder name **index** instead of **tabOne**
2. OR don't use a stack (no folder) for initialRoute.

I may be misunderstanding something.  I'm wondering if anyone has run into anything similar?

Thanks for any help.

**Comment by elritz** [comment]
In you _layout for your Tabs try writing this in the file:

```
<Tabs initialRouteName='TabOne'>
  <Tabs.Screen
    name='TabOne' 
  />

</Tabs>
```
Also what is your full route that you have to redirect to the Tabs > TabOne? If this is it " /(tabs) " then it wont work!
